src/loader/config.js

```diff
@@ -104,17 +104,24 @@ export default class Config {
 
 	/**
 	 * Adds a module to the configuration with default field values if it
-	 * doesn't exist. Otherwise, returns the module.
+	 * doesn't exist. Otherwise, throws an exception.
 	 * @param {string} moduleName
+	 * @param {object} extraProps
```

Comment:
What are extraProps? Could we maybe add an example for this?

Reply:
I'll rename it to `moduleProperties` and explain what it is in JSDoc